storage/local/interface.go

```diff
@@ -34,6 +34,9 @@ type Storage interface {
 	// from the provided Sample as those labels are considered equivalent to
 	// a label not present at all.
 	Append(*model.Sample)
+	// Throttled returns true if the Storage has too many chunks in memory
+	// already or has too many chunks waiting for persistence.
+	Throttled() bool
```

Comment:
I'm still for `Throttle` as in a question. This is a simply a "should I do X?"-request.
We had alternatives where the implication was that the storage requested anything, which I find misleading.

Or invert the bool and make it `HasCapacity()`?